src/store.rs

```diff
@@ -610,18 +628,34 @@ pub struct LevelCacheStore<E: Element> {
     // unnecessarily.
     store_size: usize,
 
+    // If provided, the store will use this method to access base
+    // layer data.
+    reader: Option<ExternalReader<R>>,
+
     _e: PhantomData<E>,
 }
 
-impl<E: Element> ops::Deref for LevelCacheStore<E> {
+impl<E: Element, R: Read + Send + Sync> fmt::Debug for LevelCacheStore<E, R> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "LevelCacheStore")
```

Comment:
should write some details of its fields here, see https://doc.rust-lang.org/std/fmt/struct.DebugStruct.html for an easy way to do this